applications/nrf5340_audio/src/bluetooth/bt_renderer/volume/bt_rend_vol_internal.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _BT_REND_VOL_INTERNAL_H_
+#define _BT_REND_VOL_INTERNAL_H_
+
+#include <zephyr/bluetooth/conn.h>
+
+/**
+ * @brief	Set volume to a specific value.
+ *
+ * @param	volume	The absolute volume to set.
+ *
+ * @return	0 for success,
+ *		-ENXIO if the feature is disabled.
+ *		Other errors from underlying drivers.
+ */
+int bt_rend_vol_set(uint8_t volume);
+
+/**
+ * @brief	Turn the volume up by one step.
+ *
+ * @return	0 for success,
+ *		-ENXIO if the feature is disabled.
+ *		Other errors from underlying drivers.
+ */
+int bt_rend_vol_up(void);
+
+/**
+ * @brief	Turn the volume down by one step.
+ *
+ * @return	0 for success,
+ *		-ENXIO if the feature is disabled.
+ *		Other errors from underlying drivers.
+ */
+int bt_rend_vol_down(void);
+
+/**
+ * @brief	Mute the output volume of the device.
+ *
+ * @return	0 for success,
+ *		-ENXIO if the feature is disabled.
+ *		Other errors from underlying drivers.
+ */
+int bt_rend_vol_mute(void);
+
+/**
+ * @brief	Unmute the output volume of the device.
+ *
+ * @return	0 for success,
+ *		-ENXIO if the feature is disabled.
+ *		Other errors from underlying drivers.
+ */
+int bt_rend_vol_unmute(void);
+
+/**
+ * @brief	Discover VCS and included services.
+ *
+ * @param	conn	Pointer for peer connection information.
+ *
+ * @note	This will start a GATT discovery and setup handles and
+ *		subscriptions for VCS and included services.
+ *		This shall be called once before any other actions related with VCS.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int bt_rend_vol_discover(struct bt_conn *conn);
+
+/**
+ * @brief	Initialize the Volume Control Service client.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int bt_rend_vol_ctlr_init(void);
```

Comment:
I remember we discussed this, but I don't remember the conclusion.
'ctlr' is correct to use here?

Reply:
yes, this is the role volume controller